go.mod

```diff
@@ -20,17 +20,17 @@ require (
 	github.com/jackc/pgconn v1.14.0
 	github.com/jackc/pgerrcode v0.0.0-20220416144525-469b46aa5efa
 	github.com/jessevdk/go-flags v1.4.0
-	github.com/lib/pq v1.10.3
-	github.com/lightninglabs/aperture v0.1.20-beta
+	github.com/lib/pq v1.10.7
+	github.com/lightninglabs/aperture v0.1.21-beta.0.20230705004936-87bb996a4030
 	github.com/lightninglabs/lightning-node-connect/hashmailrpc v1.0.2
-	github.com/lightninglabs/lndclient v0.16.0-15
+	github.com/lightninglabs/lndclient v0.17.0-0
```

Comment:
Should bump this to `v0.17.0-1` to avoid `go mod tidy` to revert `lnd` from non-RC to `.rc5`.